marsha/core/tests/test_lti.py

```diff
@@ -260,51 +277,54 @@ def test_lti_get_video_other_site_portable_ready(self, mock_verify):
         self.assertEqual(new_video.state, "ready")
         self.assertEqual(new_video.uploaded_on, video.uploaded_on)
         self.assertNotEqual(new_video.playlist, video.playlist)
-        self.assertEqual(new_video.playlist.consumer_site, other_site)
+        self.assertEqual(new_video.playlist.consumer_site, passport.consumer_site)
 
-    @mock.patch.object(LTI, "verify", return_value=True)
+    @mock.patch.object(LTIOAuthServer, "verify_request", return_value=True)
     def test_lti_get_video_other_site_several_get_latest(self, mock_verify):
-        """In case several videos exist with the same lti_id, it should duplicate the latest."""
-        ConsumerSiteFactory(name="example.com")
-        first_video = VideoFactory(
-            lti_id="df7",
-            playlist__lti_id="course-v1:ufr+mathematics+0001",
-            playlist__consumer_site__name="wrong.com",
-            playlist__is_portable_to_consumer_site=True,
-            uploaded_on=timezone.now(),
-            state="ready",
-        )
-        latest_video = VideoFactory(
+        """Above case 1-2-1-1 in case of several videos.
```

Comment:
I think I get the `1-2-1-1` sequence, but it's not that explicit. Can you develop a bit?

Reply:
Yes it's explained in the test juste above. You didn't get it in the diff but I think it's pretty clear in the code?

Reply:
My bad, you are right: it's pretty clear in the code. And I did not get the meaning of this sequence.